app/templates/components/wds-global-navigation/user-modal.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if isOpen}}
+	<div class="wds-user-modal__info">
+		{{wds-avatar src=user.avatar_url}}
+		<p class="wds-user-modal__username">{{user.username}}</p>
+	</div>
+	<div class="wds-user-modal__notifications-wrapper">
+		{{wds-on-site-notifications/notifications header=header}}
+	</div>
+	{{wds-global-navigation/sign-out-button model=user.items.[4]}}
```

Comment:
Remove API!  😄 
But seriously, I propose to change API to prevent being dependant on array index.